packages/nocodb/src/utils/nc-config/NcConfig.ts

```diff
@@ -149,7 +149,7 @@ export class NcConfig {
       port: process.env.NC_PORT,
       tryMode: !!process.env.NC_TRY,
       worker: !!process.env.NC_WORKER,
-      dashboardPath: process.env.NC_DASHBOARD_PATH,
+      dashboardPath: process.env.NC_DASHBOARD_PATH || '/dashboard',
```

Comment:
We can move this logic to NcConfig.create function instead to support this on root level.